FetchRequests.xcodeproj/project.pbxproj

```diff
@@ -1027,6 +1027,7 @@
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_STRICT_CONCURRENCY = targeted;
```

Comment:
This runs "clean".
Full concurrency checking would fail in many ways (notably `@MainActor(unsafe)` does not work).